src/core/trtserver.h

```diff
@@ -248,6 +248,11 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetServerId(
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelRepositoryPath(
     TRTSERVER_ServerOptions* options, const char* model_repository_path);
 
+// Enable or disable periodic polling on model repository in a server
+// options.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelPolling(
```

Comment:
Instead of a true/false I think we need a ServerOptionsSetModelControlMode with options None, Poll, and Explicit. None and poll mean that the models are loaded on startup. Poll means the Polling API can be used. Explicit means that the load/unload API can be used.